client/state/selectors/get-inline-support-article-blog-id.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import 'calypso/state/inline-support-article/init';
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * @param {object} state Global app state
+ * @returns {object} ...
+ */
+export default ( state ) => get( state, 'inlineSupportArticle.blogId' );
```

Comment:
Nitpick: Given the planned deprecation of Lodash in Calypso (ref. p4TIVU-9GZ-p2), this should probably be replaced with [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).